packages/cells/src/model.ts

```diff
@@ -103,6 +103,11 @@ export interface ICodeCellModel extends ICellModel {
    */
   executionCount: nbformat.ExecutionCount;
 
+  /**
+   * Whether the code cell has been edited since the last run.
+   */
+  isDirty: boolean;
```

Comment:
Should we make it `readonly`?

```suggestion
  readonly isDirty: boolean;
```

Reply:
Which could then be moved above in the interface next to the other `readonly` attributes.

Reply:
@martinRenou what do you think about this comment :point_up:

Reply:
Done in https://github.com/jupyterlab/jupyterlab/pull/10296/commits/da20aeb0ec9e48f2453df2947369178e2f02298a

Reply:
Sorry I missed this comment 👀